src/toil/test/cwl/cwlTest.py

```diff
@@ -509,6 +509,11 @@ def test_kubernetes_cwl_conformance(self, **kwargs):
     def test_kubernetes_cwl_conformance_with_caching(self):
         return self.test_kubernetes_cwl_conformance(caching=True)
 
+    @slow
+    @needs_kubernetes
+    def test_kubernetes_cwl_20(self):
+        return self.test_kubernetes_cwl_conformance(selected_tests="20", caching=False)
```

Comment:
Does this work with caching?  This only seems to be testing one case.

Maybe we could remove/replace:

```
    @slow
    @needs_kubernetes
    @pytest.mark.xfail
    def test_kubernetes_cwl_conformance(self, **kwargs):
        return self.test_run_conformance(batchSystem="kubernetes",
                                         **kwargs)
    @slow
    @needs_kubernetes
    @pytest.mark.xfail
    def test_kubernetes_cwl_conformance_with_caching(self):
        return self.test_kubernetes_cwl_conformance(caching=True)
```

With just this test and add as necessary?

Reply:
OK, I've cut the xfail tests in favor of one test that runs test 20 with and without caching.

Once we fix all the known-bad tests we have issues for, maybe the other CWL tests will Just Work when we turn them on on Kubernetes.